__tests__/Organization.js

```diff
@@ -33,302 +31,315 @@ afterAll(() => {
   nock.enableNetConnect();
 });
 
-// Responses to requests
-beforeEach(() => {
-  getMockCall('/v4/user/', userResponse);
-  getMockCall('/v4/info/', AWSInfoResponse);
-  getMockCallTimes(`/v4/organizations/${ORGANIZATION}/`, orgResponse, 2);
-  getMockCall('/v4/clusters/', v4ClustersResponse);
-  getMockCallTimes(`/v4/clusters/${V4_CLUSTER.id}/`, v4AWSClusterResponse, 2);
-  getMockCallTimes(
-    `/v4/clusters/${V4_CLUSTER.id}/status/`,
-    v4AWSClusterStatusResponse,
-    2
-  );
-  getMockCall(`/v4/clusters/${V4_CLUSTER.id}/apps/`, appsResponse);
-  // Empty response
-  getMockCall(`/v4/clusters/${V4_CLUSTER.id}/key-pairs/`);
-  getMockCallTimes(`/v4/organizations/${ORGANIZATION}/credentials/`, [], 2);
-  getMockCall('/v4/releases/', releasesResponse);
-  getMockCall('/v4/appcatalogs/', appCatalogsResponse);
-});
-
-// Stop persisting responses
 afterEach(() => {
-  expect(nock.isDone());
+  if (!nock.isDone()) {
+    console.error('Nock has pending mocks:', nock.pendingMocks());
+  }
+
   nock.cleanAll();
 });
 
-describe('Navigation', () => {
-  it('navigation has selected the right page when in organization list route', async () => {
+describe('', () => {
```

Comment:
Why does this have to be in a separate `describe` block?

Reply:
The beforeEach with all the `getMockCalls` applies to most but not all tests, so I wrapped them in a describe

Reply:
Specifically, the organization deletion tests add an organization to the organization list response manually, in order to test deleting it.

Thinking about it a bit, I could change the test so that we remove `acme` instead of the randomly generated organization we insert into the response, but I didn't want to make too much of a change to the test logic itself.

Reply:
Nevermind, I didn't see the indentation and I thought this describe block was separated from the others, and couldn't understand how it would work 😅